docs/.vuepress/config.js

```diff
@@ -37,7 +37,13 @@ export default defineUserConfig({
     plugins: {
       mdEnhance: {
         align: true,
-        include:true,
+        include: {
+          getPath: (file) => {
+            if (file.startsWith("@orgdocs"))
+              return file.replace("@orgdocs", path.resolve(__dirname, "../../node_modules/.github/"));
```

Comment:
The only thing I'd request, but isn't needed to merge, would be to wrap this in curly braces. I know they aren't required and in some environments are discouraged, but personally the visual break between elements makes things more clear. 

Just my two cents

Reply:
This has been added now